image/image.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../index.css">
+    <link rel="stylesheet" href="image.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="short icon" href="../rath_.ico">
+    <title>Doc | Image</title>
+</head>
+<body>
+    <header class="header-nav nav-root">
+        <div class="header-nav__line"></div>
+        <div class="desktop-nav">
+            <nav class = "nav">
+                <div class="desktop-nav__heading"><a href="/index.html" class="heading-link">RathUI</a></div>
+                <div class="nav-social">
+                    <a href="https://github.com/virendrawadher" class="fa fa-github fa-2x social-link"></a>
+                    <a href="https://twitter.com/virendra_wadher" class="fa fa-twitter fa-2x social-link"></a>               
+                </div>
+            </nav>
+        </div>
+        <div class="mobile-nav">
+            <div class="header-nav__line"></div>
+            <div class="mobile-menu">
+                <span class="mobile-menu__line"></span>
+                <span class="mobile-menu__line"></span>
+                <span class="mobile-menu__line"></span>
+            </div>
+            <div class="mobile-nav__heading">
+                RathUI
+            </div>
+        </div>
+    </header>
+    
+    <div class="main-content">
+        <div class="content-menu">
+            <ul class="mobile-menu__item">
+                <li><h4>Getting Started</h4>
+                    <ul class="mobile-menu__subitem">
+                        <li><a href="/installation/installation.html">Installation</a></li>
+                    </ul>
+                </li>
+                <li>
+                    <h4>Components</h4>
+                    <ul class="mobile-menu__subitem">
+                        <li><a href="/alert/alert.html">Alert</a></li>
+                        <li><a href="/avatar/avatar.html">Avatar</a></li>
+                        <li><a href="/badge/badge.html">Badge</a></li>
+                        <li><a href="/Button/button.html">Button</a></li>
+                        <li><a href="/card/card.html">Card</a></li>
+                        <li><a href="/form/form.html">Form</a></li>
+                        <li><a href="/image/image.html">Image</a></li>
+                        <li><a href="/input/input.html">Input</a></li>
+                        <li><a href="/list/list.html">List</a></li>
```

Comment:
The color contrast between the text color and background color is poor, therefore difficult to read. Maybe choose a darker grey.